config-linux.md

```diff
@@ -384,6 +384,31 @@ The following parameters can be specified to set up the controller:
     }
 ```
 
+## <a name="configLinuxVTPMs" />vTPMs
+
+**`vtpms`** (array of objects, OPTIONAL) lists a number of emulated TPMs that
+will be made available to the container.
+
+Each entry has the following structure:
+
+* **`statePath`** *(string, REQUIRED)* - full path to a directory where the vTPM is to write its persistent state into
+* **`vtpmVersion`** *(string, OPTIONAL)* - The version of TPM to emulate; either 1.2 or 2; default is 1.2
+* **`createCerts`** *(boolean, OPTIONAL)* - Whether to create certificates for the vTPM
+
+The `statePath` MUST be unique per container.
```

Comment:
nit: suggested copy edit:

> If the `vtpms` array contains duplicated entries with the same `statePath`, the runtime MUST [generate an error](runtime.md#errors).

to echo (ish) our existing language [here][1].

[1]: https://github.com/opencontainers/runtime-spec/blame/v1.0.0/config-linux.md#L45